### ISSUE TITLE
Update Licensing and copyright page

### ISSUE THREAD

**Issue by naushinthomson** [elifesciences/schematron-gitbook#173]
- [x] Update page to make licensing/permissions tagging as clear as possible with a map of how editorial should write stickies and how to convert this to xml
- [x] Becky approved page
- [x] Melissa approved page
- [x] Fred approved page
- [x] James approved page
- [x] Naushin approved page
- [x] Claudia approved page
- [x] Editorial approved page
- [x] Share page with Exeter 

### Definition of done
- [x] Team reviewed and approved page?
- [x] Add changelog to page
- [x] Assign Fred to update the Schematron if relevant

**Comment by naushinthomson** [comment]
@bcollins14 @JGilbert-eLife @FAtherden-eLife @griffithsc @Melissa37 

This page has now been updated! I've added to the [Figure adaptations and reproductions](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/article-details/content/licensing-and-copyright#figure-adaptations-and-reproductions) sections - let me know if anything could be clearer. @griffithsc I'd especially value your feedback on this as someone with fresh eyes! :D
